docs/setting-up-relay.md

```diff
@@ -0,0 +1,41 @@
+# Setting up Relay 
+
+To set up Relay, integrate a repository using Relays's web interface, and install the command-line interface (CLI).
+
+## Integrate a repository with Relay
+
+You must integrate a repository with Relay before you can add a workflow.
+
+To integrate your repository, log in to the Relay web interface at [https://app.relay.sh/](https://app.relay.sh/) and click **Integrations** > **Add Integration**.
```

Comment:
the app.relay.sh url isn't quite ready yet and we use nebula.puppet.com elsewhere. Want to just leave it as nebula.puppet.com in both places until that new subdomain is ready?